frontend/wallet/src/render/Hooks.re

```diff
@@ -6,6 +6,15 @@ let useActiveAccount = () => {
   };
 };
 
+let useToast = () => {
```

Comment:
Maybe we can add a TODO comment here pointing out that this code will behave strangely if two toasts are triggered in close proximity (the second will be cancelled early by the setTimeout of the first).